packages/compose/src/hooks/use-focus-outside/index.ts

```diff
@@ -3,58 +3,7 @@
  */
 import { useCallback, useEffect, useRef } from '@wordpress/element';
 
-/**
- * Input types which are classified as button types, for use in considering
- * whether element is a (focus-normalized) button.
- */
-const INPUT_BUTTON_TYPES = [ 'button', 'submit' ];
-
-/**
- * List of HTML button elements subject to focus normalization
- *
- * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus
- */
-type FocusNormalizedButton =
-	| HTMLButtonElement
-	| HTMLLinkElement
-	| HTMLInputElement;
-
-/**
- * Returns true if the given element is a button element subject to focus
- * normalization, or false otherwise.
- *
- * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus
- *
- * @param eventTarget The target from a mouse or touch event.
- *
- * @return Whether the element is a button element subject to focus normalization.
- */
-function isFocusNormalizedButton(
-	eventTarget: EventTarget
-): eventTarget is FocusNormalizedButton {
-	if ( ! ( eventTarget instanceof window.HTMLElement ) ) {
-		return false;
-	}
-	switch ( eventTarget.nodeName ) {
-		case 'A':
-		case 'BUTTON':
-			return true;
-
-		case 'INPUT':
-			return INPUT_BUTTON_TYPES.includes(
-				( eventTarget as HTMLInputElement ).type
-			);
-	}
-
-	return false;
-}
-
 type UseFocusOutsideReturn = {
-	onFocus: React.FocusEventHandler;
-	onMouseDown: React.MouseEventHandler;
-	onMouseUp: React.MouseEventHandler;
-	onTouchStart: React.TouchEventHandler;
-	onTouchEnd: React.TouchEventHandler;
```

Comment:
These were all necessary to catch focus events that happened within the container in order to cancel the timeout. Since we're not cancelling a timeout on focus, we don't need any of these or the code that supports them.